### ISSUE TITLE
Performance Lab: Branding

### ISSUE THREAD

**Issue by dainemawer** [WordPress/performance#144]
## Performance Lab: Branding
As briefly discussed in our [Weekly Performance Meeting from 1st February 2022](https://make.wordpress.org/core/2022/02/01/performance-team-meeting-summary-1-february-2022/)

@felixarntz brought up an item with regards to design some assets for the "Performance Lab" plugin.
As we're no stranger to opinion in the WP community, it would be great to get everyones ideas / thoughts / opinions in terms of what they would like to see in terms of creative options.

Some initial ideas / inspiration from my side:

1. Jetpack Boost has a nice feel to it: https://wordpress.org/plugins/jetpack-boost/ - I also like the fact that it has a catch phrase.
2. Coincidently, this Dribbble Shot: https://dribbble.com/shots/16485339-Axelerated-Software kinda incorporates the JetPack logo, with WordPress.org colours which is actually really nice.
3. We could also go with some a bit more 3D: https://dribbble.com/shots/17317377-Athens-Medica - 3D is trending quite wildly across the web at the moment, with more skeumorphic design making a come back. 

Considering this plugin will eventually land up in core, I think retain the default WordPress.org colours is a must, with a nice logon / monogram could be quite effective.

Lets get the discussion going :)

**Comment by dainemawer** [comment]
Spitballed a couple of ideas this morning:

#### Logomark with 3D Element
![3d-logo](https://user-images.githubusercontent.com/48082535/152304106-60479faa-3b3b-43b8-b68d-f12925d6088f.png)

#### Logon 1
![logon-logo](https://user-images.githubusercontent.com/48082535/152304126-cee2eaa0-0378-4b83-83d1-bb679a08a332.png)

#### Logon 1 with Illustration
![logon-logo-with-illustration](https://user-images.githubusercontent.com/48082535/152304124-d125b3f0-ab24-4521-9778-b19dff21a539.png)

#### Logon 2
![logon-2](https://user-images.githubusercontent.com/48082535/152304288-8aec2951-795b-4491-a4ac-29819c972a98.png)

#### Logon 3
![logon-3](https://user-images.githubusercontent.com/48082535/152304118-6c831cc1-f573-4884-a8dc-ec2d0322d155.png)

#### Logon 4
![logon-4](https://user-images.githubusercontent.com/48082535/152304122-6147c78b-b454-4426-b03b-66918c7854ea.png)

**Comment by felixarntz** [comment]
@dainemawer Awesome! I personally like your "Logon 1" and "Logon 4", but curious to hear what others think. A few general points of feedback:

* I'm not sure we should include a tagline like you have throughout. It would require discussion on what it should be, but most importantly it makes me think too much of this as marketing a product, which this plugin isn't really. _If_ we want to include some sort of tagline, I'd even prefer something less "catchy" and more descriptive/objective, e.g. "Feature plugins to boost performance" (probably there's something better though).
* In any case, for the banner, I think a version with some sort of illustration looks more appealing than just the logo part on white.
* We'll also need a square image for the plugin icon - which I assume we could just take which ever logo (just the logo without the text) that we decide on.

**Comment by jeffpaul** [comment]
In general I tend to shy away from text within the banner (or icon) graphics as they are inherently not translatable, so you could end up with a fr.wordpress.org/plugins/slug with the readme in French but the text on the banner in English.

**Comment by felixarntz** [comment]
Good point @jeffpaul, yeah that's another reason to not include a tagline. I think the plugin name would be the only exception to that, since it doesn't (or at least shouldn't) get translated anyway.

**Comment by javiercasares** [comment]
Like the Logon 4.

**Comment by mxbclang** [comment]
I also like 1 and 4, slightly leaning toward 4!

**Comment by dainemawer** [comment]
Duly noted all :) 

- @felixarntz - happy to take the tagline out. I noticed JetPack Boost had one, so thats where I got the inspiration from. 
- I can play with some kind of illustration / treatment once we decide on a direction, totally agreed.
- Noted on the plugin icon and all that - theres good documentation around this all.

**Comment by adamsilverstein** [comment]
I like 1 & 4 as well, I like how 1 has a round outline/shape which will work well at small sizes (icon).

**Comment by AymenLoukil** [comment]
I like logon 4

**Comment by erikyo** [comment]
![performance-Mv5 1](https://user-images.githubusercontent.com/8550908/155849670-3fd7dec2-a130-445c-a473-d91cf1277208.png)

The lightning bolt might be a bit of an overused icon (caching, amp...) but i want to give my contribution too. It might be nice to add the lightning emoji in the name of the plugin like "Performance ⚡"

@dainemawer - I played with your first design adding a triangle to create the shape of the lightning bolt and then overlapped the trapeziums above to give it the a folded look. Honestly I like the material design I don't know had catched that 😅. in any case, to do it right you should make a paper model or a small study on the shape because, as @adamsilverstein said, it must be recognisable even when reduced to a 32x32 icon.

In addition, I wanted to draw attention to the most appropriate palette/colours to use. I think that turquoise or red or orange/yellow (which I used) are perfect to express the meaning of this plugin. But unfortunately they are colours that have little contrast on bright colors and probably using them on a white background does not make them shine. With the colour I have chosen the complementary is the electric violet which is not bad, also because it is a plugin "to go fast" and must have a certain vivacity...

**Comment by sudeep510** [comment]
Hi Everyone,

Wanted to contribute with a few ideas I tried out today, hope I’m not too late..
Would love to hear your thoughts on this

Here are a few Logo ideas, a bit of play with the P and L letters

**Logo 1**

![logo-variation 1](https://user-images.githubusercontent.com/72914363/152417111-9a7b7187-e764-4c17-9189-c365c29ccce2.png)


**Logo 2**

![logo-variation 2](https://user-images.githubusercontent.com/72914363/152417212-1ebba48b-51be-4c5d-889e-ec1109109498.png)


**Logo 3**

![logo-variation 3](https://user-images.githubusercontent.com/72914363/152417246-43e5a970-bbd0-422f-8d73-d31fad601c5c.png)


And here are a few Banners - tried using more performance-related elements to help convey the purpose


**Banner 1**

![performance-lab-bannerv1](https://user-images.githubusercontent.com/72914363/152417306-e76f4780-383a-404d-94f3-91886e7b09c4.png)


**Banner 2**

![performance-lab-bannerv2](https://user-images.githubusercontent.com/72914363/152417337-ccd77bdd-5193-43ca-b08b-178f0d103099.png)


**Banner 3**

![performance-lab-bannerv3](https://user-images.githubusercontent.com/72914363/152417384-7b35ff00-a721-4cb2-bcde-5d6d87a3eb12.png)


**Banner 4**

![performance-lab-bannerv4](https://user-images.githubusercontent.com/72914363/152417437-e5d3116c-1288-4553-b650-2a56511bc076.png)

**Comment by javiercasares** [comment]
> The lightning bolt might be a bit of an overused icon (caching, amp...) but i want to give my contribution too. It might be nice to add the lightning emoji in the name of the plugin like "Performance ⚡"

Like it a lot :)

**Comment by ThierryA** [comment]
Thanks all for sharing ideas, great to see some discussions happening. I will add my two cents.

Before jumping into designs, I would suggest to brainstorm a bit further some "brand principals" such as:

- What we want the branding to transpire?
- What we **don't** want the branding to transpire?
- Some descriptive words about the look and feel we are after?
- Somethings else?

Something to take in consideration is that we are not building a "competitive product" (which is very difference from existing performance plugins). With that in mind, my suggestions would be the following:

1. The look and feel should be subtle and humble
2. It should try to somehow transpire the fact that it is a set of WordPress Core feature plugins, so fairly WordPressy like styling
3. The branding should not come across as yet another new competing performance plugin nor should it come across as "selling"

Regarding timeline, I think we should allow ourselves to take the time we need, without overthinking it indeed. We could totally rollout v1 without branding (for example for the [XML Sitemap feature plugin](https://wordpress.org/plugins/core-sitemaps/) we never had branding)

As another point of reference, the [REST API feature plugin](https://wordpress.org/plugins/rest-api/) didn't have branding for a while and eventually came up with something but fairly subtle.

Would love to hear more thoughts :)

**Comment by felixarntz** [comment]
+1 to @ThierryA's feedback. The branding shouldn't invoke the impression that you activate this plugin and your WordPress site will be fast. Stating that would be overselling it, and the purpose of this plugin is explicitly not that - of course we aim to improve performance in the WordPress ecosystem, but the plugin is a collection of performance-focused "feature plugins" for WordPress core. The plugin is not going to be a suite of all crucial performance features you need to make your site fast - that's where the existing performance plugins have their market, and the Performance Lab plugin should indeed not falsely raise an impression of wanting to compete with them. It's a feature plugin collection, which also makes it somewhat of a beta testing plugin.

I also recommend we look at previous feature plugins to come up with some simple branding, like the [REST API feature plugin](https://wordpress.org/plugins/rest-api/). Potentially we could have some nice type of "lab icon" and just the "Performance Lab" title next to it. What I'd strongly recommend against is having any kind of tagline. It makes our plugin seem like it's a product that wants to sell the user on something.

I would say we should focus our branding on the "lab" part to the same degree we focus on the "performance" part of the plugin name - most of the proposals so far focus on "performance", but not the "lab" aspect of it.

**Comment by erikyo** [comment]
Sorry, I didn't catch that at all... I prepared a new sketch with a flask icon that imho gives the idea of the laboratory, to which I added the lightning bolt icon (so that the logo contains both concepts). 
I made it this way to be as minimalist as the wp rest api banner, what do you think?

![performance-v2-04](https://user-images.githubusercontent.com/8550908/152647758-cf9c6577-ff8f-4f15-b49a-45d8d90d9246.png)

edit: if someone want "fork" the design [here you are](https://modul-r.codekraft.it/wp-content/uploads/2022/02/lab.svg) (font used Montserrat)

**Comment by JustinyAhin** [comment]
I mostly agree with I generally agree with @ThierryA's points above. Because it's a feature plugin, which will eventually end up in Core, we should keep the branding simple. So IMO, an icon with the plugin name next to it will do.

For the icon itself, we could either go with the initials of the plugin, like @dainemawer's proposal 4 or @sudeep510's proposal 1.

**Comment by sudeep510** [comment]
Hi Everyone,

After going through the inputs provided here, have attempted another version to be in line with the thoughts conveyed for the branding we would like to opt for.

Here are the ideas behind these designs:

- The thought behind this logo variation was to keep it simple and focus more on associating with a “Labs” element and continue playing with the letter P for performance
- As the plugin will eventually be a part of WordPress core, first preference has been given to using WordPress colors. However, an alternate version with different colors too has been showcased.
- As we don’t wish to add any tagline and convey any wrong promises, have kept the banner simple and subtle. Apart from having the logo and the plugin name close together, I have added some extra elements so that the design doesn’t look too plain.

**Main Logo**

![performance-lab-primary-logo](https://user-images.githubusercontent.com/72914363/152817288-215e6004-ccd8-49be-9943-019b3f1639c2.png)                       ![performance-lab-primary-logo-2](https://user-images.githubusercontent.com/72914363/152818275-e02fe05e-409d-4501-b3b3-27e23dbe21aa.png)


**Banner variation 05**

![performance-lab-banner1](https://user-images.githubusercontent.com/72914363/152817453-fd153f9a-946d-4799-984f-7bd005ca10d0.png)

**Banner variation 06**

![performance-lab-banner2](https://user-images.githubusercontent.com/72914363/152817503-00dd324a-dd5a-43e8-827c-9dfa51dc96d4.png)

**Banner variation 07**

![performance-lab-banner3](https://user-images.githubusercontent.com/72914363/152817534-2515127b-decc-4631-bd29-55c0388d0915.png)

**Banner variation 08**

![performance-lab-banner4](https://user-images.githubusercontent.com/72914363/152817573-0915aa35-5459-43c3-9354-0377bf8f6826.png)

For ease of viewing - if anyone would like to view all the variations I shared together along with more information about the design properties, you can check out the [Figma file here](https://www.figma.com/file/uhMiLOprXXPnwkQYalOjNY/Performance-Lab-Logo%2FBanner?node-id=0%3A1)

**Comment by dainemawer** [comment]
Some further simplifications and plugin logos.

**Logo 1**
![Frame 20](https://user-images.githubusercontent.com/48082535/153123701-1524617d-3394-4a6b-9948-10c1f0b1c9f0.png)

**Logo 4**
![Frame 21](https://user-images.githubusercontent.com/48082535/153123708-2da329eb-cc91-440b-8673-b6a5e287f6ce.png)

**New Idea**
![Frame 22](https://user-images.githubusercontent.com/48082535/153123709-ae38cc88-f6a0-4c3f-b1f9-bcdff29544e1.png)

**Comment by felixarntz** [comment]
@sudeep510 I personally like your first main logo variation from https://github.com/WordPress/performance/issues/144#issuecomment-1031619957. One question there: What is the shade of blue you're using there? Could you maybe create a version that uses the exact same blue that the WordPress logo uses? I'd be curious to see a version like that.

For the banner variation, maybe something like your banner variation 7, but I think we should keep it even simpler and not display the plugin name on it, since it won't be accessible, and the name will be showed in another place next to the banner anyway. I think that text could simply be removed from the banner. For reference, see other feature plugins like https://wordpress.org/plugins/shiny-updates/ that also don't have any title in their banner.

**Comment by sudeep510** [comment]
Hi @felixarntz The black and blue version of the logo is already using colors from [WordPress branding](https://make.wordpress.org/design/handbook/design-guide/foundations/colors/). Here are a few more variations: 

**[With WordPress logo colors:](https://make.wordpress.org/marketing/handbook/resources/style-guide-and-brand-book/#color-palette)**

![logo colours](https://user-images.githubusercontent.com/72914363/154726065-ae08448f-398d-4171-a33c-60bd70264cae.png)


**[With WordPress brand colors:](https://make.wordpress.org/design/handbook/design-guide/foundations/colors/#blues)**

![brand colours](https://user-images.githubusercontent.com/72914363/154726176-3f185e35-02fa-42d7-b6e6-782a67c630bb.png)


For the banner, I have removed the title and kept it simple. I have kept the colors as they were. I will change the color based on the above logo variation that we go with.

![performance-lab-banner](https://user-images.githubusercontent.com/72914363/154726266-15d3908d-ac48-449e-85fa-7d170aaa937c.png)

Let me know your thoughts.

**Comment by erikyo** [comment]
@sudeep510 my suggestion is to move the center of the cruet to make it perfectly aligned with the center. Something like this... what do you think? (first-your version with P internal circles shape reduced, second your version with alembic circle shape expanded, third is the proposal with the centered cruet)
![PPP](https://user-images.githubusercontent.com/8550908/154852953-831a60ad-bb71-4f8c-9c82-ec7f47c2089e.png)

**Comment by sudeep510** [comment]
Thanks @erikyo for the suggestions. I will make the changes before sharing the final design files, once the design is approved.

**Comment by ThierryA** [comment]
Great to see proposals flowing on this issue, thanks to everyone involved so far. A few other folks have showed interest in contributing to the branding exercise so I encouraged them to chime in this issue and also have the opportunity share their creative approach/questions, design etc. as well.

**Comment by rwlands** [comment]
Thanks @ThierryA and hey everyone. Myself and a couple colleagues are designers and we're keen to collaborate and see how we can contribute to the community. We've been looking at your concepts and taken some inspiration for our own explorations. We should have some concepts to share in the next couple days. We'd love to take your feedback and see if we can help with creative input.

**Comment by rwlands** [comment]
Hi Everyone, 

So this week we focused on exploration, getting some inspiration and settling on three distinct directions we take the branding in. Next week we want to refine down to our best ideas and take on your feedback so we can take a bit more of an exploration for colour / wordmark and how the logo would work on the plugins page and other uses. 

We wanted to show you work in progress and check if we're getting closer to something that could work.

Any feedback would be appreciated, I suppose we'd be keen to hear from you:

1. Is there a preference from these three directions which we should explore more next week?
2. Is there a particular concept inside the direction that stands out to you? Something you prefer most and least?
3. Anything you'd like us to try or make sure we avoid doing?

There will be more opportunity to give feedback and select between more polished options next week.

Cheers,
Tom

<img width="1920" alt="PL WiP – 1" src="https://user-images.githubusercontent.com/100207191/155763267-649ecdab-ea56-465a-8682-0e9a4bcdd097.png">


<img width="1920" alt="PL WiP – 2" src="https://user-images.githubusercontent.com/100207191/155763271-f2976d01-23dd-408b-811a-8df24865c9f1.png">


<img width="1920" alt="PL WiP – 3" src="https://user-images.githubusercontent.com/100207191/155763275-f8e16f15-4660-49e5-9d2b-3c513f8d724f.png">

**Comment by ThierryA** [comment]
Thanks for share the progress of your concepts 😄 

> Is there a preference from these three directions which we should explore more next week?

I have a preference for the "Connecting the dots" concepts.

> Is there a particular concept inside the direction that stands out to you? Something you prefer most and least?

"Concept 2.0" and "Sketch Ideation"

> Anything you'd like us to try or make sure we avoid doing?

In an idea world, banding should try to somehow transpire the fact that it is a set of "official" WordPress Core feature plugins. With that said, we can perhaps explore using WP colors in future iterations. I wouldn't say that it is a top priority, I like the more modern "pastel" color too and as long as it doesn't come across as competing performance plugin and remains "humble/minimal" branding it should be ok.

**Comment by dainemawer** [comment]
I quite likely the folding paper direction. Though, I do agree as the project is quite tightly-coupled to WordPress Core that it should be more closely aligned with the WordPress branding in general.

**Comment by ThierryA** [comment]
Figured it would be a good idea to explain the rational of my preference. To me the "Connecting the dots" is the least shiny branding and the one we can iterate the most on to get closer to a "WordPress Core" like feel (especially given its flat nature). It is also quite modern and humber IMO, which is great! It will also be great for things like swag printing, playing with contrast etc.

**Comment by rwlands** [comment]
Thanks for the feedback so far, if anyone wanted to discuss or add any other thoughts please do and we'll start working on refined options from today. I'm happy to join todays weekly call if that helps.

It's clear from discussions that we want to stay closer to the WP brand colours - we'll iterate some versions of those. 

So far we can see preference toward:
- Folding paper 
- Connecting dots

(Excuse the designer labels!)

How does everyone think about refining down to just 2 options based on these routes?

**Comment by ThierryA** [comment]
Thanks Tom. More refining on the 2 options above SGTM.

> I'm happy to join todays weekly call if that helps

Weekly meeting are happening on chat in the Performance [Slack channel](https://wordpress.slack.com/archives/C02KGN5K076). @bethanylang let's add an agenda item to call this issue out for feedback.

**Comment by mxbclang** [comment]
@ThierryA Done!

@rwlands [More details on joining the WordPress.org Slack here](https://make.wordpress.org/chat/). Note that it requires [creating a wordpress.org account first](https://login.wordpress.org/register). Let me know if you have any trouble.

**Comment by omarreiss** [comment]
I like the moving form top left concept 3. I would prefer a somewhat generic and flat branding that uses WordPress color palette. I feel most of these last iterations float away from the WordPress branding a little too much for my taste.

**Comment by eclarke1** [comment]
My preference is the 'Connecting the dots' sketch ideation approach, but my preference would also be towards WordPress colour palette - this one feels more simple to me.
(My second favourite is 'Moving form' sketch ideation approach but see we are trying to refine down to 2 options)

**Comment by felixarntz** [comment]
I'd prefer using the WordPress color palette as well. My preference is the "Connecting the dots" approach too, second favorite "Moving form". So I guess I'm completely on the same page as @eclarke1  here 😆 

I also would much prefer that we use "flat" colors and no gradients, at least for the logo mark itself.

**Comment by rwlands** [comment]
Thanks again everyone for sharing your feedback. There's enough of a consensus for us to keep iterating towards a refined smaller set of options. We'll aim to share those with you tomorrow. 

Since the latest polls/votes have come in it looks to be most preferences are with:
 - Connecting the dots
 - Moving form 

Look forward to showing you how we progress this.

**Comment by rwlands** [comment]
Hey there, just wanted to present back the logos with the updated 'Wordpress colours' added in. 

How do people feel about the colour options? 

Seeing the new colours are we able to pick out preferences for the logo mark? Personally I lean towards Connecting dots - Option C and Moving form Option A,B. 

Today we're going to mockup these against the banners, merch and with the word mark as well. We should be able to present final options back to you by end of the day.


![PL WiP – 4](https://user-images.githubusercontent.com/100207191/156541002-a82e6565-e780-4b7c-9221-a2b1bd316ad2.jpg)
![PL WiP – 5](https://user-images.githubusercontent.com/100207191/156541047-23438aeb-3c99-4f20-8ba8-47794b8b0f48.jpg)

**Comment by ThierryA** [comment]
Thanks for the new round of design, my preference in order:

1. Connecting the dot - A2 (I did also like the Shape of the P in the previous round which changed in this round)
2. Connecting the dot - B2
3. Moving form  - A2
4. Moving form  - C2

WordPress Color Palette also have the dark grey, would be great to see a variant with it (maybe grey and white for example)?

Looking forward to hear other folks thoughts and can't wait to see it along side the typography for "Performance Lab".

**Comment by adamsilverstein** [comment]
I'm also a fan of A2, I like the "P" shape and color.

**Comment by tillkruss** [comment]
I love the moving form ones 🥰

**Comment by felixarntz** [comment]
My personal favorites in order:
1. Moving form - B2
2. Connecting the dots - A2
3. Moving form - A2

**Comment by eclarke1** [comment]
I think me and @felixarntz are nearly one person 😆 ! My personal favourites are:

1. Moving form - B2
2. Connecting the dots - A2
3. Connecting the dots - B2

**Comment by mxbclang** [comment]
Favorites in order:

1. Connecting the Dots – A2
2. Moving form – B2
3. Moving form – B3

I really like the colors for the black background versions, but they don't read as "WordPress" to me, where the blue background versions do.

**Comment by tillkruss** [comment]
My personal favorites in order:
1. Moving form - B2
2. Moving form - B1
3. Moving form - B3

**Comment by sudeep510** [comment]
Hi all, 
I worked on a few more variations for the logo. I stuck with WordPress colors and the inspiration for this is based on the preferences and ideas shared in the above comments.

![Frame 1](https://user-images.githubusercontent.com/72914363/156621517-84408fa6-a71d-434a-9804-756afc8205f0.png)


Would really appreciate your suggestions on these please and maybe we could also consider teaming up with other designs to arrive at something solid.

**Comment by ThierryA** [comment]
Thanks for sending some more variation, Sudeep! My preference out of the batch you share would be v1 (although spacing and the balance feel a bit off). The others (IMO) all have spacing/balance and look and feel flaws. Also in terms of putting the logo in different contrasts it may be challenging with the 3 color logo as such.

> consider teaming up with other designs to arrive at something solid

+1, your feedback/preference and suggestion (similar to what Erik did in previous comments) on the other design iterations shared by Tom would also be super useful.

**Comment by eclarke1** [comment]
Personally I feel the new logos added here https://github.com/WordPress/performance/issues/144#issuecomment-1058314653 are drifting slightly further away from the simplistic forms shown here https://github.com/WordPress/performance/issues/144#issuecomment-1057876398

Out of all these together, I still prefer the simpler forms shown in:

1. Moving form - B2
2. Connecting the dots - A2

**Comment by swissspidy** [comment]
"Connecting the dots - A2" really caught me eye. I think it would work very well, even when changing colors depending on context. So +1 from me on that one.

**Comment by rwlands** [comment]
Thanks everyone for your input. 

Looks like we've got some clear front runners:

Connecting the dots - A2 (7 votes)
Moving form - B2 (5 votes)

Looking forward to showing you our final concepts.

**Comment by rwlands** [comment]
Hey all, just in time for the weekend. Wanted to share with you our final designs for the end of this week. 

We wanted to start by saying we're very conscious that others have also put forward their own concepts in the thread, these have served as our inspiration - so credit to those of you. 

We've selected 3 concepts and included the typographic lockup, and shown how it could come together with the banner in the WP Plugin page.  Concept 3 is the 'bonus' option we wanted to include alongside the two highest voted.

Two things to note: 

- We've included a strapline in the banner, as some of you mentioned showing the logo and word mark seemed a bit overkill right next to the title and icon. The Strap line should be seen as Lorem Ipsum, it's our attempt at sample copy framing the plugin. We'll leave that out of our scope.

- The banner uses shaped drawn from the logo, we've intentionally been stripped back on design or illustration because of the minimal branding we're working with and we don't want it to look like a product plugin (promising too much).

It would be great to reach a consensus on one preferred option from the three concepts next week. We'll then package up our work and share files with you so you can take it forward.

Wishing you all well.

![PL WiP – 32](https://user-images.githubusercontent.com/100207191/156814226-5966fd77-7746-4da6-ac18-58a484a23fec.jpg)
![PL WiP – 29](https://user-images.githubusercontent.com/100207191/156814202-6d9078cb-2a0a-4dfa-8892-8b1b44125765.jpg)
![PL WiP – 30](https://user-images.githubusercontent.com/100207191/156814216-bbdcfed0-dde9-4f39-bf5e-9bf1862931c8.jpg)

**Comment by erikyo** [comment]
I really like the third one, but I would love it with a logo colored (and not black, may even a colored outline could be cool)! But please note, [the payoff is not recommended because it cannot be translated](https://github.com/WordPress/performance/issues/144#issuecomment-1029085065)

**Comment by mxbclang** [comment]
I honestly can't decide if I like 1 or 2 better! I think I'm leaning _slightly_ toward 2.

**Comment by felixarntz** [comment]
@rwlands Thank you for these iterations!

My personal favorite is concept 2, followed by concept 1.

One general point of feedback that would apply to either: For the banner design, it would be great if we could remove the text there entirely and leave it to containing only shapes and the logo mark. Putting text in the banners is discouraged for accessibility reasons (it's impossible to put an alternative text for the image), and it's worth noting that the banner always appears next to the plugin name, so the context would be clear.

Regarding the banner in concept 2 specifically, I see that one doesn't contain the logo mark - maybe it's better like that since in that design the logo is not some form of a "P", but if the consensus ends up being concept 2, I'd be curious to see an alternative version of the banner that also includes the logo (e.g. like in the other two banner designs).

**Comment by ThierryA** [comment]
I have a **strong** preference for concept 1 (and 3). I can see how we could do cool stuff/variation with the "P" branding like the the outline in concept 3, nice for swag, banners etc. and personally feel that the dot/bubble branding is very common. I also think that concept 2 icon will be more challenging to use in various context, for example on a white (or other color) bg unlike the P concepts which will be more versatile due to it single tone.

For some reason at some point we lost the [concept.2.0](https://github.com/WordPress/performance/issues/144#issuecomment-1051077516) variation of the P.

What I would love to see (if possible indeed) is:  concept 1, 2, 3 and concept.2.0 (all with the blue color palette) without text on the banner to be able to compare apple with apple

Then, it would also be great to put concept 1, 2, 3 and concept.2.0 in various contrast, for example white icon on WP dark grey, blue icon on white bg and/or else to give an idea of what it looks like when it is used beyond the plugin page (even better on some sticker, t-shirt, banners or what ever but I understand this may be stretching the ask).

**Comment by dainemawer** [comment]
I quite like Concept 2 - I think its stronger in visual communication around "performance". Both "P" concepts to me don't really tie back to WordPress all that much, except for the colours. The "experimental" bubbles, to me, feel like an addition to Dashicons, which I think is a nice coincidence :)

**Comment by rwlands** [comment]
Thanks for the feedback all. No clear stand out amongst the options yet. Options 2 is slightly ahead of option 1. So we could take those forward into the final exploration.

@ThierryA in response to your request we've shown concept 2.0 now as well. we should try narrow our vote between option 1,3,concept 2.0 - for second place.

Although many WP plugins do use EN text in their banners, we can appreciate the translation and accessibility concerns. We've left open the space in the banner for either illustration or the word mark. If that needs more work we can keep that out of scope until we've selected the logo / word mark option.

To keep the ball rolling we'll spend today mocking up the final use cases and await any feedback. Thanks again for sparing your time to take a look.

![PL-Concept1](https://user-images.githubusercontent.com/100207191/157238776-810e043d-9c91-40ef-8ab8-3bc7c65a8750.jpg)
![PL-Concept2](https://user-images.githubusercontent.com/100207191/157238784-6f3df80b-06f4-4ea4-ab19-1435308f275d.jpg)
![PL-Concept3](https://user-images.githubusercontent.com/100207191/157238799-5d4aa522-4ebb-4e42-bced-99e7b6769d5f.jpg)
![PL-Concept4](https://user-images.githubusercontent.com/100207191/157238809-cd6ef826-14a1-4dc6-a9c8-e91e405df299.jpg)

**Comment by mxbclang** [comment]
I still prefer concept 2, but if we're committed to using a "P," I like concept 1 best.

**Comment by ThierryA** [comment]
Thanks for putting these 4 concepts together, my vote in order:

1. Concept 1
2. Concept 3
3. Concept 4
4. Concept 2

@bethanylang I suggest to socialize the latest iteration during the meeting today to get as many vote as possible.

**Comment by swissspidy** [comment]
I also think that bubbles like in concept 2 are rather common, so my vote is for concept 1 still.

1. Concept 1
2. Concept 4
3. Concept 3
4. Concept 2

**Comment by jeffpaul** [comment]
Too many cooks in the kitchen here, let's rely on a core committer's opinion and just ship Concept 1 so the released plugin at least has some design on the respective WP.org page.

**Comment by pbearne** [comment]
Concept 1
Concept 4
Concept 3
Concept 2

**Comment by jamesozzie** [comment]
Great work all. My own preferences:
- Concept 1
- Concept 4
- Concept 3
- Concept 2

**Comment by tillkruss** [comment]
- Concept 1
- Concept 2
- Concept 4
- Concept 3

**Comment by felixarntz** [comment]
1. Concept 1
2. Concept 2
3. Concept 4
4. Concept 3

**Comment by zero4281** [comment]
1. Concept 1
2. Concept 2
3. Concept 4
4. Concept 3

**Comment by adamsilverstein** [comment]
I like Concept 1 the most!

**Comment by mitogh** [comment]
Concept 1 has my vote as well :+1:

**Comment by dainemawer** [comment]
1. Concept 2
2. Concept 1
3. Concept 4
4. Concept 3

**Comment by rwlands** [comment]
Thanks everyone the votes are in! and we've got enough to determine so we'll close votes.

We're excited to share the finished concept soon, hopefully today or tomorrow.

**Comment by rwlands** [comment]
Hey everyone - we wanted to share the final work on the brand in some more detail.

Our mech mockups are limited to a couple of examples at the moment, maybe someone wants to try their hand at their own merch? We might add a few more next week.

We've opened up a Figma file which includes the assets and guidelines to the identity we designed: [Click here](https://www.figma.com/file/6DNarP3c7WHHh1AF64G0yp/Performance-Lab-%2F-Brand-Guidelines?node-id=1%3A72)

You can view the guidelines online through this prototype: [Click here](https://www.figma.com/proto/6DNarP3c7WHHh1AF64G0yp/Performance-Lab-%2F-Brand-Guidelines?page-id=1%3A66&node-id=1%3A72&viewport=241%2C48%2C0.15&scaling=scale-down-width)

**Comment by mitogh** [comment]
Thanks @rwlands looks like the file is not being shared or does not exist:

> Either this file doesn’t exist or you don’t have permission to view it. Ask the file owner to verify the link and/or update permissions.

When opening the file, or maybe I need an account to view the file?

**Comment by ThierryA** [comment]
Thanks @rwlands, at the moment I am also getting an error when opening the links. Looking forward to seeing it though 😊 Feel free to share it as images to this issue as well as "preview" for those who are not clicking on the links.

**Comment by jeffpaul** [comment]
@rwlands on top of fixing the Figma link access, posting the images here or a direct link to them so we can get them added to the plugin would be great.  To confirm, the filename/size requirements are:
- banner-772x250.png
- banner-1544x500.png
- icon-128x128.png
- icon-256x256.png
- icon.svg

Similarly, a 1280×640px designed image would be ideal to add to this GitHub repo for cases where someone shares this repo on social that a custom OpenGraph image will be used instead of the [generically created GitHub one](https://github.blog/2021-06-22-framework-building-open-graph-images/).

**Comment by rwlands** [comment]
Hi all, 

So the issue with Figma access should be resolved now. 

Have uploaded a zip file to wetransfer with all the asset requests @jeffpaul 
The brand guidelines are also in added as PDF.
https://we.tl/t-46rY8rzTal

Thanks a lot all for your feedback to get us here.

**Comment by ThierryA** [comment]
_Edit: oops seems like Tom beat me to it by 2 mins lol, thanks Tom!_

Figma files ["Brand Guidelines"](https://www.figma.com/file/6DNarP3c7WHHh1AF64G0yp/Performance-Lab-%2F-Brand-Guidelines?node-id=1%3A66) and ["Design Assets"](https://www.figma.com/file/6DNarP3c7WHHh1AF64G0yp/Performance-Lab-%2F-Brand-Guidelines?node-id=1%3A66) appear to be working now. What an amazing result (IMO), super grateful for your contributions @rwlands and co 🙏 .

> posting the images here or a direct link to them so we can get them added to the plugin would be great

I extracted the following from the figma files, is that what you were expecting @jeffpaul ?

Banner 1880x608 (png)

![Banner_1880x608](https://user-images.githubusercontent.com/3581402/157867098-a6a7ea33-21d8-41ec-a9b3-509c11ad60c6.png)

Banner 772x250 (png)
![Banner_772x250](https://user-images.githubusercontent.com/3581402/157867111-85acd08d-6dd3-4bd6-bb14-17fa6f9a94bf.png)

Banner 1544x500 (png)
![Banner_1544x500](https://user-images.githubusercontent.com/3581402/157867114-cb97fe39-761b-45b7-ad54-5f1fb4f629ee.png)

Icon 256x256 (png)
![Icon_256x256](https://user-images.githubusercontent.com/3581402/157867129-a7bd02ab-dd88-459e-98f9-b8bbb74986b8.png)

Icon 128x128 (png)
![Icon_128x128](https://user-images.githubusercontent.com/3581402/157867132-01c2b51c-4084-40e6-955e-c14c16c16217.png)

Icon 256x256 (svg)
![Icon_256x256](https://user-images.githubusercontent.com/3581402/157867133-41de2e2d-277c-4eee-b598-695283e3e983.svg)

Also loving the swag exploration (I will try to organize some for WCEU, I will float a request form) 🎉 
![Performance Lab – Brand Guidelines – 11](https://user-images.githubusercontent.com/3581402/157867865-2f090c10-0756-4f31-9382-a299fee4d196.png)

**Comment by jeffpaul** [comment]
@rwlands I'm seeing the banner files are one pixel too tall (e.g. 772x251 instead of 772x250) and while I could trim the files I'd hate to trim from somewhere you might choose not to, so hoping you could check out the banner images and trim them down to match 1544x500 and 772x250 perhaps?

**Comment by rwlands** [comment]
@jeffpaul Let me sort that out and reshare.
@jeffpaul I'll point you to the zip file with all the assets - https://we.tl/t-46rY8rzTal (these are the correct source rather than those added as well by Thierry). For quick use, I've added the banner here.
![Banner_772x250](https://user-images.githubusercontent.com/100207191/158157952-a744af56-a30f-4345-abc9-b77aa435f7e0.png)

**Comment by jeffpaul** [comment]
@rwlands the banner images in that ZIP still seem to show as 1544x501 (instead of 1544x500) and 772x251 (instead of 772x250).  Perhaps you shared the wrong link or forgot to update the new PNG files with trimmed dimensions in that ZIP?

**Comment by felixarntz** [comment]
@rwlands Confirming that I also see the banner images from your latest ZIP in https://github.com/WordPress/performance/issues/144#issuecomment-1066577374 being 1px too tall. Also, the other banner size which is supposed to be 1880x608 is entirely different, that image is actually 2706x876.

**Comment by jeffpaul** [comment]
For what its worth I'm unaware of any need for a 1880x608 or 2706x876 banner image so its possible those could be removed.

**Comment by rwlands** [comment]
Looks like there was a transparent line of pixels causing the issue. Updated here - https://we.tl/t-HGznK6Gb6V

**Comment by felixarntz** [comment]
Thanks @rwlands, these look all good to me, dimensions are fixed now. 🎉 

Will open a PR to include the banner and icon here so that they will be deployed to wordpress.org alongside the next release.

**Comment by felixarntz** [comment]
See #231 for the PR adding these assets for deployment to wordpress.org.

**Comment by felixarntz** [comment]
Fixed via #231 - we now have branding for our plugin, which will be deployed to the wordpress.org repository on Monday with the `1.0.0-beta.2` release! 🎉 

Kudos again to @rwlands and all the other contributors who helped make this happen.
